src/components/App/App.js

```diff
@@ -4,35 +4,42 @@ import './App.css';
 // import movieData from '../../movieData';
 import MoviesContainer from '../MoviesContainer/MoviesContainer';
 import MovieDetails from '../MovieDetails/MovieDetails';
+import Nav from '../Nav/Nav';
 import { fetchMovies } from '../../apiCalls';
-import { fetchMovieInfo } from '../../apiCalls';
+// import { fetchMovieInfo } from '../../apiCalls';
 import { Route } from 'react-router-dom';
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: [],
+      movieLibrary: [],
+      displayedMovies: [],
+      searchBar: '',
```

Comment:
Do you think we should perhaps make the Nav component a class component and move this searchBar propery into Nav's state? This way we could be sticking to the controlled component format we have already learned about...thinking about keeping state where it belongs.